JavaScript: Binary Search Tree Project: add `prettyPrint()` function translated from Ruby's BST project (#24540)
TheOdinProject/curriculum: javascript/computer_science/project_binary_search_trees.md

1. Print out all elements in level, pre, post, and in order

</div>

> **Tip:** If you would like to visualize your binary search tree, here is a `prettyPrint()` function that will `console.log` your tree in a structured format. This function will expect to receive the root of your tree as an argument.

Reviewer: Wording suggestion: Just being extra specific on how to pass in the root.

```suggestion
**Tip:** If you would like to visualize your binary search tree, here is a `prettyPrint()` function that will `console.log` your tree in a structured format. This function will expect to receive the root of your tree as the value for the `node` parameter.
```
